### ISSUE TITLE
Metamask signature doesn't work (pop up closing)

### ISSUE THREAD

**Issue by hugoobauer** [ChainSafe/dappeteer#93]
Hello !
I can't manage to sign a transaction. The Metamask popup is closing right away (like a flash) after I clicked on "Connect Wallet" without any error message. Even if I comment the .click() and do it by my self on Chromium, I have the same issue. I don't know if I'm doing something wrong or if it is a bug. Any help is appreciated.

Here my code sample : 

```
import puppeteer from "puppeteer";
import * as dappeteer from '@chainsafe/dappeteer';

(async () => {
    const browser = await dappeteer.launch((puppeteer as any), {
        metamaskVersion: 'v10.8.1',
    });

    const metamask = await dappeteer.setupMetamask(browse);
    await metamask.addNetwork({
        networkName: 'Binance Smart Chain',
        rpc: 'https://bsc-dataseed.binance.org/',
        chainId: 56,
        symbol: 'BNB',
        explorer: 'https://bscscan.com'
    });
    await metamask.switchNetwork('Binance Smart Chain');
    await metamask.addToken('0x00e1656e45f18ec6747f5a8496fd39b50b38396d');

    const page = await browser.newPage();
    await page.setViewport({
        width: 1200,
        height: 1000
    });
    await page.goto('https://app.bombcrypto.io/webgl/index.html');
    await page.screenshot({path: 'screenshot1.png'});
    const game = await page.$('html body div#unity-container.unity-desktop');
    const box1 = await game?.boundingBox();
    if (box1) {
        await page.waitForTimeout(6000);
        await page.screenshot({path: 'screenshot2.png', clip: box1});
        await page.mouse.click(box1.x + (box1.width / 2), box1.y + 500);
        await page.waitForTimeout(10000);
        // await metamask.approve();
        // await page.bringToFront();
        // await page.waitForTimeout(5000);
        // await page.screenshot({path: 'example6.png', clip: box1});
        // await metamask.confirmTransaction();
        // await metamask.sign();
        // await page.waitForTimeout(10000);
    }

    // await browser.close();
})();
```
```
  "dependencies": {
    "@chainsafe/dappeteer": "^2.4.0",
    "puppeteer": "^13.2.0",
    "puppeteer-extra": "^3.2.3",
    "puppeteer-extra-plugin-stealth": "^2.9.0",
    "typescript": "^4.5.5"
  },
  "devDependencies": {
    "@types/puppeteer": "^5.4.4"
  }

```

**Comment by roman-raizen** [comment]
@hugoobauer same issue. Did you figure it out? Metamask popup just closes right away.

**Comment by mzahirr** [comment]
Is there any progress for popup closed issue ?

**Comment by mpetrunic** [comment]
should be fixed in 3.0.0-rc.0
